src/translations/en.json

```diff
@@ -300,7 +300,7 @@
         },
         "announce_volume_strategy": {
             "label": "Volume strategy for Announcements",
-            "description": "When an announcement is being broadcast to this player, how should the volume be adjusted temporary (use in combination with the volume value below).",
+            "description": "When an announcement is being broadcast to this player, how should the volume be adjusted temporarily (use in combination with the volume value below).",
```

Comment:
If you're going to fix the grammar here, you probably want a question mark on the end of the line at the same time.

Reply:
I said grammar not punctuation! But yeah I will fix that and some other things :-) Thanks!